homeassistant/components/fortios/device_tracker.py

```diff
@@ -47,49 +47,49 @@ def get_scanner(hass, config):
         return None
 
     status_json = fgt.monitor("system/status", "")
-    fos_major_version = int(status_json["version"][1])
+    version = status_json["version"][1:]
+    _LOGGER.debug("FortiOS version: %s", version)
 
-    if fos_major_version < 6 or fos_major_version > 7:
+    fos_major, fos_minor, fos_patch = (int(x, 10) for x in version.split("."))
+
+    if (
+        fos_major < 6
```

Comment:
Please use `AwesomeVersion` to compare version. It's our goto tool for version checking and is available as a core requirement.

https://github.com/ludeeus/awesomeversion#example-usage

Reply:
Thanks @MartinHjelmare  I have pushed new code using AwesomeVersion.